### PR TITLE
Adds logging to admin use of Smite/Bless

### PR DESCRIPTION
**Pull request by Kyep**
Admin usage of smite/bless is now logged. All other admins (plus adminlog) can see it.

No CL, admin only, no difference in functionality, just logging, etc.

### CODE REVIEW

File: code/modules/admin/topic.dm [hunk 1]
```diff
@@ -1744,30 +1744,39 @@
...
 			if("Super Powers")
 				var/list/default_genes = list(REGENERATEBLOCK, NOBREATHBLOCK, COLDBLOCK)
 				for(var/gene in default_genes)
 					H.dna.SetSEState(gene, 1)
 					genemutcheck(H, gene,  null, MUTCHK_FORCED)
 					H.update_mutations()
+				logmsg = "superpowers."
+		if(logmsg)
+			log_admin("[key_name(src.owner)] answered [M]'s prayer with a blessing: [logmsg]")
+			message_admins("[key_name_admin(src.owner)] answered [M]'s prayer with a blessing: [logmsg]")
```
Comment (Krausus) [inline code/modules/admin/topic.dm]: The targeted mob should use a keyed name in the logs, like the admin does.

File: code/modules/admin/topic.dm [hunk 2]
```diff
@@ -1787,43 +1796,55 @@
...
 				evilcookie.bitesize = 100
 				H.drop_l_hand()
 				H.equip_to_slot_or_del(evilcookie, slot_l_hand)
+				logmsg = "a hellwater cookie."
 			if("Gib")
+				logmsg = "gibbed."
 				M.gib(FALSE)
+		if(logmsg)
+			log_admin("[key_name(src.owner)] answered [M]'s prayer with a smiting: [logmsg]")
+			message_admins("[key_name_admin(src.owner)] answered [M]'s prayer with a smiting: [logmsg]")
```
Comment (Krausus) [inline code/modules/admin/topic.dm]: Keyed names here, too.

File: code/modules/admin/topic.dm [hunk 3]
```diff
@@ -1775,8 +1775,8 @@
 					H.update_mutations()
 				logmsg = "superpowers."
 		if(logmsg)
-			log_admin("[key_name(src.owner)] answered [M]'s prayer with a blessing: [logmsg]")
-			message_admins("[key_name_admin(src.owner)] answered [M]'s prayer with a blessing: [logmsg]")
+			log_admin("[key_name(src.owner)] answered [key_name_admin(M)]'s prayer with a blessing: [logmsg]")
```
Comment (tigercat2000) [inline code/modules/admin/topic.dm]: use key_name in log_admin, not key_name_admin (the former contains anchor links that look nasty in the logs)

**Comment by Kyep** [comment]
@tigercat2000 Done

File: code/modules/admin/topic.dm [hunk 4]
```diff
@@ -1744,30 +1744,39 @@
...
 			if("Super Powers")
 				var/list/default_genes = list(REGENERATEBLOCK, NOBREATHBLOCK, COLDBLOCK)
 				for(var/gene in default_genes)
 					H.dna.SetSEState(gene, 1)
 					genemutcheck(H, gene,  null, MUTCHK_FORCED)
 					H.update_mutations()
+				logmsg = "superpowers."
+		if(logmsg)
+			log_admin("[key_name(src.owner)] answered [key_name(M)]'s prayer with a blessing: [logmsg]")
+			message_admins("[key_name_admin(src.owner)] answered [key_name_admin(M)]'s prayer with a blessing: [logmsg]")
```
Comment (Fox-McCloud) [inline code/modules/admin/topic.dm]: No implied `src` here or otherwise.

**Comment by Kyep** [comment]
@Fox-McCloud Changed

**Pull request merged**